src/mugs.js

```diff
@@ -1464,6 +1464,22 @@ define([
                     return data.id && data.id.slice(data.id.lastIndexOf("/") + 1);
                 }
             },
+            labelItext: {
```

Comment:
itext-related code typically lives in the javaRosa plugin. What do you think of moving this [there](https://github.com/dimagi/Vellum/blob/6e234ddf7ee1a162e8a65da516fe549d60a2e752/src/javaRosa/plugin.js#L567-L584)? Also, does this `validationFunc` override the [default one for `labelItext`](https://github.com/dimagi/Vellum/blob/6e234ddf7ee1a162e8a65da516fe549d60a2e752/src/javaRosa/plugin.js#L802)?